src/components/Image/schema.ts

```diff
@@ -13,6 +13,15 @@ const ImageBase = {
     },
 };
 
+const StyleBase = {
+    type: 'object',
+    additionalProperties: false,
+    required: [],
+    properties: {
+        backgroundColor: {type: 'string'},
```

Comment:
Here could be any css property, not only backgroundColor
maybe `string`

Reply:
added a few widespread options: `width`, `height`, `color`